compiler/rustc_codegen_llvm/Cargo.toml

```diff
@@ -8,11 +8,15 @@ test = false
 
 [dependencies]
 # tidy-alphabetical-start
+ar_archive_writer = "0.4.0"
```

Comment:
Adding this isn't necessary, right?

Reply:
Oops, that was left over from my experimentations.